### ISSUE TITLE
Start with (very) first digit

### ISSUE THREAD

**Issue by flinx42** [jrhunger/2600-digits#7]
Should start with *3* or at least not make it an error if you choose that as first digit

**Comment by jrhunger** [comment]
I will see if i can make it optional.

**Comment by jrhunger** [comment]
What if i seeded 3.14 at the top and people had to go from there?

**Comment by flinx42** [comment]
So you want 90% of the population to fail the game immediately?

**Comment by jrhunger** [comment]
yes?  this is definitely a game for a niche audience.  I figure anyone who might play it would recognize 3.14 and realize they need to continue the sequence.
And millions of digits of pi are a just google away.

**Comment by flinx42** [comment]
Fair enough. Personally, I'd still want to start with the 3, as that was my initial expectation.
I'll raise two more issues now though.
